File: Without Node Module/Xampp/htdocs/core/week-1/c-programs/programs/42.php
<?php
    $matrix1 = [[1, 2, 3], [4, 5, 6], [7, 8, 9]];
    $matrix2 = [[9, 8, 7], [6, 5, 4], [3, 2, 1]];
    $resultMatrix = [];

    for ($i = 0; $i < count($matrix1); $i++) {
        for ($j = 0; $j < count($matrix2[0]); $j++) {
            $resultMatrix[$i][$j] = 0;
            for ($k = 0; $k < count($matrix1[0]); $k++) {
                $resultMatrix[$i][$j] += $matrix1[$i][$k] * $matrix2[$k][$j];
            }
        }
    }

    echo "Matrix Multiplication:<br>";
    printMatrix($resultMatrix);

    function printMatrix($matrix) {
        foreach ($matrix as $row) {
            echo implode(", ", $row) . "<br>";
        }
    }
?>

File: One Task/Task 1/c programming/42.c
// Matrix Multiplication

#include <stdio.h>

int main() {

    int m1[3][3] = {
        {1, 2, 2},
        {3, 0, 4},
        {4, 1, 2}
    };
    
    int m2[3][3] = {
        {3, 4, 1},
        {4, 2, 2},
        {3, 2, 1}
    };

    int ans[3][3];

    for(int i=0; i<3; i++) {
        for(int j=0; j<3; j++) {
            ans[i][j] = 0;
            for(int k=0; k<3; k++) {
                ans[i][j] += m1[i][k] * m2[k][j];
            }
            printf("%2d ", ans[i][j]);
        }
        printf("\n");
    }

    return 0;
}